### PR TITLE
Improved comments, added millis() macro, made example main() more useful for systick_irq example

### PR DESCRIPTION
**Pull request by ADBeta**

### CODE REVIEW

**Review by prosper00 — COMMENTED** [review]
what do you think about switching the GPIO stuff to funPinMode() and funDigitalWrite()?

**Comment by ADBeta** [comment]
Now uses funPinMode etc, tested to work the same

File: examples/systick_irq/systick_irq.c [hunk 1]
```diff
@@ -2,74 +2,81 @@
...
  * 05-12-2023 C. Lohr (Modified to reflect updated sysclk)
+ * 09-25-2024 ADBeta (Minor updates to main loop, comments and added
+ *                    convinient macro function)
  */
 
 #include "ch32v003fun.h"
 #include <stdio.h>
 
-volatile uint32_t systick_cnt;
+#define SYSTICK_CMP_1_MILLISECOND ((uint32_t)FUNCONF_SYSTEM_CORE_CLOCK / 1000)
```
[the remark this answers is not in the document]
Comment (ADBeta) [reply examples/systick_irq/systick_irq.c:12]: I hope no one ever uses this chip at a HCLK of 1KHz
Usually 48MHz is default so this should be fine, and anyone going below 1000 will have issues more important that this breaking I would think

[the remark this answers is not in the document]
Comment (TommyMurphyTM1234) [reply examples/systick_irq/systick_irq.c:12]: I assumed it was an edge case - if even possible at all - but just wanted to ask just in case. :-)

**Comment by ADBeta** [comment]
Typos are fixed

File: examples/systick_irq/systick_irq.c (hunk 1 above)
[the remark this answers is not in the document]
Comment (ADBeta) [reply examples/systick_irq/systick_irq.c:12]: no problem
I'm not sure if it is possible, but if it is it's probably an awful idea anyway haha.

**Comment by TommyMurphyTM1234** [comment]
> Typos are fixed

I think I'me looking at the latest changes but still see typos... :-|

**Comment by ADBeta** [comment]
Double checked, fixed indents and one missing fix -
Previous fixes were visible on the previous commit, not sure why you couldn't see them :thinking:

**Comment by TommyMurphyTM1234** [comment]
> Double checked, fixed indents and one missing fix - Previous fixes were visible on the previous commit, not sure why you couldn't see them 🤔

I don't understand - when I go into the PR and review the latest changes I still see this for example:

**Comment by ADBeta** [comment]
> > Double checked, fixed indents and one missing fix - Previous fixes were visible on the previous commit, not sure why you couldn't see them 🤔
> 
> I don't understand - when I go into the PR and review the latest changes I still see this for example:
> 
> 

I missed that one somehow....
I've fixed that one too now, should all be done :crossed_fingers: 
Let me know if you know of a good spellchecker plugin for nvim/vim lol

**Comment by cnlohr** [comment]
This all looks good to me now!

**Pull request merged**